test/metabase/driver/postgres_test.clj

```diff
@@ -347,14 +348,6 @@
 
 ;;; ----------------------------------------- Tests for exotic column types ------------------------------------------
 
-(deftest ^:parallel json-query-support-test
-  (testing "JSON database support options behave as they're supposed to"
-    (are [details expected] (= expected
-                               (driver/database-supports? :postgres :nested-field-columns {:details details}))
-      {}                      true
-      {:json-unfolding true}  true
-      {:json-unfolding false} false)))
```

Comment:
I removed this test because of changes to `database-supports?`. `json-unfolding-default-test` now covers it.